example_new/Config.json

```diff
@@ -93,142 +93,15 @@
             "osteoarthritisknee"
         ],
         "interventions": {
-            "active_type_id": null,
+            "active_type_id": "simple",
             "types": {
                 "simple": {
                     "active_period": {
                         "start_time": 2022,
-                        "finish_time": 2022
+                        "finish_time": null
```

Comment:
Would it be clearer to represent "no finish time" with a missing key, rather than having it be `null`?